src/utils.jsx

```diff
@@ -0,0 +1,8 @@
+import axios from "axios";
+
+const apiClient = axios.create({
+  baseURL: "https://dhiran-news.onrender.com/api",
+  timeout: 5000,
+});
+
```

Comment:
Nice, but this file should also have functions that also house the requests themselves and send back the data in the right format